osc/core.py

```diff
@@ -6378,6 +6378,39 @@ def get_buildconfig(apiurl, prj, repository, path=None):
     f = http_GET(u)
     return f.read()
 
+# This is always replacing a possible exiting config for now
+# we could extend the _pbuild file easily, but what should we do with multiple instances of the _config?
+def create_pbuild_config(apiurl, project, repository, arch, project_dir):
+    # get expanded buildconfig for given project / repository
+    bc = get_buildconfig(apiurl, project, repository)
+    if bc == None:
+        e.osc_msg = 'failed to get build config for project \'%s\' repository \'%s\'' % (project, repository)
+        raise
+
+    file = open(project_dir + '/_config', 'w')
+    file.write(decode_it(bc))
+    file.flush()
+
+    # create the _pbuild file based on expanded repository path informations
+    pb = ET.fromstring('<pbuild></pbuild>')
+    tree = ET.ElementTree(pb)
+    preset = ET.SubElement(pb, 'preset', name=repository, default="") # default should be empty, but ET crashes
+    bi_text = decode_it(get_buildinfo(apiurl, project, '_repository', repository, arch, specfile="Name: dummy"))
+    root = ET.fromstring(bi_text)
+# cross compile setups are not yet supported
+#    for path in root.findall('hostsystem'):
+#        ET.SubElement(preset, 'hostrepo').text = path.get('url')
+    for path in root.findall('path'):
+        ET.SubElement(preset, 'repo').text = path.get('url')
+
+    ET.SubElement(preset, 'arch').text = arch
+    try:
+        ET.indent(tree, '  ')
+    except:
```

Comment:
lgtm checker already complained: `1 for Except block handles 'BaseException'`
Could you specify the exception type?